docs/release-notes/v1.50-v1.59.md

```diff
@@ -10,6 +10,224 @@
+
+* ??? abstract "Important considerations to delete Reference Genomes"
+    * Only users assigned as Administrators can execute the <code>DELETE /api/v1/manage-data/data</code> endpoint to delete data, including reference genomes
+    * If the Reference Genome is linked to the VCF group, then the reference genome should be unlinked. As a reference genome is required for filtering by genes, variantFeature parameter is expected to not work for the VCF group without a reference genome
+    * In case the selected object has more than one version, the endpoint deletes all of them
+    * If a Reference Genome is linked to any other data and the Reference Genome is deleted, the endpoint will also delete the linked data associated with it then searching for variant by variantFeature (genes) won't be available for linked VCF groups.
+
+* ??? danger "Endpoint to delete Reference Genomes"
+    * DELETE /api/v1/manage-data/data
+
+![ManageData.png](1.59/ManageData.png)
+<figcaption>The user can now remove reference genomes if no longer needed. The endpoint <code>DELETE /api/v1/manage-data/data</code> now also includes the option to delete Reference Genome (accession number is required)</figcaption>
+
+* **Improved Support for Files (formerly known as Attachments): Delete Files via API**
+
+Admins can now manage imported files through the <code>DELETE /api/v1/manage-data/data</code> endpoint (**manageData**), improving the control over file lifecycle management.
+
+* ??? abstract "Important considerations"
+    * Manage organization AND Access all data permissions are required.
+    * The whole stack of versions of the mentioned attachment is to be deleted.
+
+* ??? danger "Endpoints to delete files"
+    * DELETE /api/v1/manage-data/data
+
+![DeleteFile.png](1.59/DeleteFile.png)
+<figcaption>The endpoint <code>DELETE /api/v1/manage-data/data</code> has been enhanced, allowing the deletion of (attached) files from ODM</figcaption>
+
+#### [All Users]
+
+* **Updates in the Study Menu interface: Menu accessible on both sides of the menu**
+
+The Study Menu now offers enhanced access options, making it easier to navigate ODM. Users can access the menu from both the top and right sides of the metadata editor, streamlining data management workflows
+
+![Menu.png](1.59/Menu.png)
+<figcaption>Updated view of the metadata editor. The menu can easily be accessed from the top part of the study, or by clicking on the right side, next to the Export button</figcaption>
+
+* **Updates in the Study Menu interface: File Manager**
+
+The dashboard now includes a dedicated File Manager accessible from the sidebar menu. This feature lets users view and filter files by type, such as Reference Genome, Flow Cytometry group, Library, and Sample Metadata.
+
+* ??? abstract "Important considerations"
+    * Filters will be displayed in ascending order
+    * If no filters are applied, the types selected as default will be Reference Genomes and Dictionaries.
```

Comment:
This is not entirely correct: if no filters are applied, all types will be displayed, but by default Reference Genomes and Dictionaries types are selected
![image](https://github.com/user-attachments/assets/c175deb4-3ba7-4ee2-a2ed-b7b9a9ec271b)